HumanControls/JoystickAxis.cpp

```diff
@@ -27,23 +30,34 @@ double JoystickAxis::calcResult(int rawValue)
 {
     double value = rawValue;
 
-    if (value >= m_min && value >= (m_range + m_min))
+    //General calculations
+    double deadMin = m_center - m_halfDeadZoneSize;
+    double deadMax = m_center - m_halfDeadZoneSize;
```

Comment:
Should this be `m_center + m_halfDeadZoneSize`?

Reply:
It's testing if the value is between 0 and 1023 in our case. I am not sure if it is necessary but I put it in.